rosetta/scripts/extract-patches.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd $SCRIPT_DIR
```

Comment:
Maybe let's not jump into this folder, but use `pushd-popd` instead?
```
pushd $SCRIPT_DIR
<the rest code>
popd
```

Reply:
pushd/popd isn't needed here b/c I only enter one directory and this is in a subshell so the calling parent shell doesn't change dirs